<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="300baeb7-77b7-49ff-aaa4-f940b8899e95">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/store/settings.py</remarks>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://192.168.100.234:5432/store_db</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="store_db@192.168.100.234" uuid="9f613f1a-eff6-46fc-8282-f4274fe868e6">
      <driver-ref>postgresql</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.postgresql.Driver</jdbc-driver>
      <jdbc-url>jdbc:postgresql://192.168.100.234:5432/store_db</jdbc-url>
    </data-source>
  </component>
</project>